src/TraceEvent/Parsers/ProcessMetadataTraceEventParser.cs

```diff
@@ -0,0 +1,683 @@
+﻿//<autogenerated/>
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Text;
+using Microsoft.Diagnostics.Tracing;
+using Address = System.UInt64;
+
+#pragma warning disable 1591        // disable warnings on XML comments not being present
+
+// This code was automatically generated by the TraceParserGen tool, which converts
+// an ETW event manifest into strongly typed C# classes.
+namespace Microsoft.Diagnostics.Tracing.Parsers
+{
+    using Microsoft.Diagnostics.Tracing.Parsers.ProcessMetadataEventSource;
+
+    [System.CodeDom.Compiler.GeneratedCode("traceparsergen", "2.0")]
+    public sealed class ProcessMetadataEventSourceTraceEventParser : TraceEventParser
```

Comment:
I realized above I said we didn't need to block on the name, but that probably also means we need to avoid putting the name in new public API surface. Can we switch to internal for now or should we just try to sort out the naming right away?

Reply:
I'm fine but I don't have an opinion, I think it should be the .net name.